app/services/admin/update_induction_period.rb

```diff
@@ -18,7 +18,24 @@ def initialize(author:, induction_period:, params:)
     def update_induction_period!
       previous_start_date = induction_period.started_on
       previous_end_date = induction_period.finished_on
+
       induction_period.assign_attributes(params)
+
+      # Transition induction programme to the new training programme values
+      if Rails.application.config.enable_bulk_claim
+        induction_programme = ::PROGRAMME_MAPPER[params[:training_programme]]
+        # 1. Set database enforced induction_programme but only for new inductions
+        induction_period.assign_attributes(induction_programme:) if induction_period.induction_programme.blank?
+        # 2. Set database enforced induction_programme but only for new inductions
+        # induction_period.assign_attributes(induction_programme:) unless induction_programme.eql?('unknown')
+        # 3. Set database enforced induction_programme on every edit
+        # induction_period.assign_attributes(induction_programme:)
```

Comment:
@joe-harrison-dfe @emily-prudence-dfe @peteryates 

This is up for a prod review.  Are we happy with the value we populate the old field with after enabling the feature? Both these fields use enums and our existing one enforces not being empty, so we have to give it something, and we have a few options.

cc @cpjmcquillan

Reply:
@peteryates what's your flava?

@emily-prudence-dfe proposed a new explicit entry in the old induction programme enum, and rather than mapping in the old value, just set it to either `unknown` (currently avaliable) or `post_2025_type_change` (or similar)